off-chain/src/HTTP/Client.purs

```diff
@@ -3,17 +3,100 @@ module Seath.HTTP.Client where
 import Prelude
 import Type.Proxy
 
+import Aeson (class EncodeAeson)
+import Data.Either (Either)
 import Effect (Effect)
+import Effect.Aff (Aff)
+import Payload.Client (ClientError, Options)
 import Payload.Client as Client
 import Payload.Client.ClientApi (class ClientApi)
+import Payload.Headers (Headers)
+import Payload.ResponseTypes (Response)
 import Payload.Server as Payload
+import Payload.Spec (GET, POST, Spec)
 import Seath.HTTP.Handlers as Handlers
+import Seath.HTTP.Spec (LeaderRoutes)
 import Seath.HTTP.Spec as Spec
-import Seath.Network.Types (UserNode)
+import Seath.HTTP.Types (IncludeRequest, JSend, UID)
+import Seath.Network.Types
+  ( ActionStatus
+  , GetStatusError
+  , IncludeActionError
+  , UserNode
+  )
 import Seath.Test.Examples.Addition.Types (AdditionAction)
 
-mkUserClient =
+mkUserClient
+  :: forall a
+   . EncodeAeson a
+  => Proxy a
```

Comment:
I added the proxy to be able to provide type information to the functions that only need to use the handlers without the `a` type, in that context the compiler won't get any clue about the type of `a` and we would need to pass it by hand.